src/components/BadgeSelect.tsx

```diff
@@ -91,6 +98,13 @@ class BadgeSelect extends React.Component<BadgeSelectProps, BadgeSelectState> {
 							<Text>{item}</Text>
 						</ButtonWrapper>
 					))}
+
+				{!!this.props.extraSuffix &&
+					this.props.extraSuffix.map(item => (
+						<ButtonWrapper key={item} onClick={() => this.setSelected(item)}>
+							<Text>{item}</Text>
+						</ButtonWrapper>
```

Comment:
Might be nice to have a light separator between prefix/suffix items?

Reply:
ack, good idea 👍

Reply:
@pimterry Here's what I've come up with
![screen shot 2018-02-09 at 16 11 37](https://user-images.githubusercontent.com/15064535/36037211-f79bba16-0db3-11e8-94cf-4b2c4a0308f4.png)